portfolio/src/main/webapp/script.js

```diff
@@ -60,11 +71,18 @@ function closeConnectModal(){
   modal.style.display = "none";
 }
 
+function closeQuoteModal(){
+  const modal = document.getElementById("myQuoteModal");
+  modal.style.display = "none";
+}
+
 window.onclick = function(event) {
-  const modal = document.getElementById("myIntroModal");
-  const modal2 = document.getElementById("myPictureModal");
-  const modal3 = document.getElementById("myConnectModal");
-  if (event.target === modal) modal.style.display = "none";
-  if (event.target === modal2) modal2.style.display = "none";
-  if (event.target === modal3) modal3.style.display = "none";
+  const introModal = document.getElementById("myIntroModal");
+  const pictureModal = document.getElementById("myPictureModal");
+  const quoteModal = document.getElementById("myQuoteModal");
+  const connectModal = document.getElementById("myConnectModal");
+  if (event.target === introModal) introModal.style.display = "none";
```

Comment:
Sorry didn't catch this before, but you should have braces for all control structures according to the style guide

https://google.github.io/styleguide/jsguide.html#formatting-braces-all

Also thanks for being proactive and making the variable names clearer :)

Reply:
Actually is okay to leave as-is. It's safer to have braces though.

`Exception: A simple if statement that can fit entirely on a single line with no wrapping (and that doesn’t have an else) may be kept on a single line with no braces when it improves readability. This is the only case in which a control structure may omit braces and newlines.`